src/main/java/mate/academy/springbootwebgreqit/controller/BookController.java

```diff
@@ -48,7 +48,7 @@ public BookDto createBook(@Valid @RequestBody CreateBookRequestDto requestDto) {
 
     @PutMapping("/{id}")
     @PreAuthorize("hasRole('ADMIN')")
-    public BookDto updateBook(@RequestBody UpdateBookRequestDto updateBookRequestDto) {
+    public BookDto updateBook(@RequestBody @Valid UpdateBookRequestDto updateBookRequestDto) {
```

Comment:
```suggestion
    public BookDto updateBook(@PathVariable Long id, @RequestBody @Valid UpdateBookRequestDto updateBookRequestDto) {
```

you provide @PutMapping("/{id}") path variable, but didn't get and use it